src/activity_handlers.cpp

```diff
@@ -1902,7 +1902,7 @@ void activity_handlers::train_finish( player_activity *act, player *p )
         const Skill &skill = sk.obj();
         std::string skill_name = skill.name();
         int old_skill_level = p->get_skill_level( sk );
-        p->practice( sk, 100, old_skill_level );
+        p->practice( sk, 100, old_skill_level + 2 );  // third argument is the cap it can be raised to
```

Comment:
But what are the meanings of the two parameters? And what is the meaning of the `sk` parameter in the above `get_skill_level` call? And so on.

Please don't comment the meaning of a parameter of a function call. This is redundant. If somebody wants to know this, they can look up the function documentation, which provides that information at a *single* point.